Userland/Libraries/LibWeb/Forward.h

```diff
@@ -172,6 +172,7 @@ class StaticNodeList;
 class StaticRange;
 class Text;
 class TreeWalker;
+class AccessibilityTreeNode;
```

Comment:
Please move this further up so it's alphabetically sorted with the other `class`es.